src/asmcnc/apps/drywall_cutter_app/screen_config_filechooser.py

```diff
@@ -0,0 +1,524 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is registering as "changed", even though you made changes and then undid them.
Git history might get screwed up, so I would recommend checking this file out from master (once dwt_config_loader is merged) onto this branch, to ensure that nothing gets undone/overwritten etc. by surprise. 
https://stackoverflow.com/questions/2364147/how-to-get-just-one-file-from-another-branch